ui/src/components/shared/SearchResults.vue

```diff
@@ -62,6 +61,7 @@ interface Props {
   showFilters?: boolean;
   showQuickTypeFilters?: boolean;
   quickTypeFiltersAllowed?: string[];
+  defaultQuickTypeFilter?: string;
```

Comment:
Confused why this exists. Should either set quickTypeFiltersAllowed or use the built in defaults...